src/Microsoft.Extensions.DependencyModel/project.json

```diff
@@ -14,7 +14,7 @@
   },
   "frameworks": {
     "net451": { },
-    "dotnet5.4": {
+    "dnxcore50": {
```

Comment:
This is blocking Mvc from consuming this package. I'm going to try hand editing it for now, but dotnet5.x should be the way to go.